### DISCUSSION TITLE
Memungkinkan kah menggunakan element html ?

### DISCUSSION THREAD

**Discussion by andes2912** [kirimwa-id/forum#82] [Tanya Jawab] [answered]
Halo mas @rioastamal, apakah saya bisa menggunakan element html example <br> dll pada penulisan pesan nya. Saat ini api sudah berjalan pada system development saya, namun pada saat pengiriman pesan nya saya ingin menyisipkan beberapa element html.

Terima Kasih

**Comment by rioastamal** [comment, marked as answer]
WhatsApp tidak mendukung HTML. Untuk melakukan formatting teks gunakan aturan yang sudah disediakan WhatsApp di https://faq.whatsapp.com/general/chats/how-to-format-your-messages/?lang=fi
